Lib/defconQt/glyphView.py

```diff
@@ -1359,15 +1664,12 @@ def __init__(self, glyph, parent=None):
         self._showMetricsTitles = True
 
         self.setBackgroundBrush(QBrush(Qt.lightGray))
-        self.setScene(GlyphScene(self))
+        self.setScene(GlyphScene(self, self._getSceneItems('scene-added-items')))
```

Comment:
I'm not  a huge friend of objects down in the hierarchy using object interfaces further on top. It's bad for refactoring and portability. Thus, you inject them with explicit interfaces.

This is used to delete the items the scene adds, it's a reference to the "registry" list. The scene only adds items to the list, it does not use them.

see this comment of yours: https://github.com/trufont/trufont/blob/master/Lib/defconQt/glyphView.py#L1426

> ```
            # XXX: discriminate better
            if isinstance(item, QGraphicsPathItem) and item.zValue() == -997:
                scene.removeItem(item)
```